src/transactions/feed/queryHelper.ts

```diff
@@ -384,6 +384,9 @@ export const TRANSACTIONS_QUERY = gql`
         ...TokenTransferItemV3
         ...NftTransferItemV3
         ...TokenExchangeItemV3
+        ...EarnDepositItem
+        ...EarnWithdrawItem
+        ...EarnClaimRewardItem
```

Comment:
one side effect of starting to query these items is that, even if the gate is off, we'll still be getting and displaying rewards. Maybe that's ok, but just wanted to call it out. cc @silasbw

Reply:
In testing we are already displaying them currently just as miscategorized swaps and a payment received.

#### Example from Main
<img src="https://github.com/valora-inc/wallet/assets/26950305/abc10e0d-d7e1-4f55-9282-a1752ab622f6" width="auto" height="400" />